src/renderer/screens/settings/sections/Accounts/Currencies.js

```diff
@@ -2,34 +2,40 @@
 import React, { useState, useCallback } from "react";
 import { useTranslation } from "react-i18next";
 import { useSelector } from "react-redux";
-import type { CryptoCurrency, TokenCurrency } from "@ledgerhq/live-common/lib/types";
+import type { Currency, CryptoCurrency, TokenCurrency } from "@ledgerhq/live-common/lib/types";
 import { cryptoCurrenciesSelector } from "~/renderer/reducers/accounts";
 import TrackPage from "~/renderer/analytics/TrackPage";
 import SelectCurrency from "~/renderer/components/SelectCurrency";
 import Box from "~/renderer/components/Box";
 import { SettingsSectionBody as Body, SettingsSectionRow as Row } from "../../SettingsSection";
 import CurrencyRows from "./CurrencyRows";
 import Track from "~/renderer/analytics/Track";
+import { currencySettingsDefaults } from "~/renderer/reducers/settings";
 
 export default function Currencies() {
   const { t } = useTranslation();
   const currencies = useSelector(cryptoCurrenciesSelector);
-  const [currency, setCurrency] = useState(() => {
-    const btc = currencies.find(c => c.id === "bitcoin");
-    return btc || currencies[0];
-  });
+  const [currency, setCurrency] = useState<CryptoCurrency | TokenCurrency | typeof undefined>();
 
   const handleChangeCurrency = useCallback(
-    (currency: CryptoCurrency | TokenCurrency) => {
+    (currency?: CryptoCurrency | TokenCurrency) => {
       setCurrency(currency);
     },
     [setCurrency],
   );
 
-  return !currency ? null : (
-    <Box key={currency.id}>
-      <TrackPage category="Settings" name="Currencies" currencyId={currency.id} />
-      <Track onUpdate event="Crypto asset settings dropdown" currencyName={currency.name} />
+  const currencyId = currency?.id ?? "placeholder";
+  const currencyName = currency?.name ?? "placeholder";
+
+  const isCurrencyDisabled = useCallback(
+    (currency: Currency) => !currencySettingsDefaults(currency).confirmationsNb,
```

Comment:
maybe check for at least a key in the object maybe there are currencies with a single setting that isn't confirmationNb :thinking:

Reply:
I've used the current logic here.

https://github.com/LedgerHQ/ledger-live-desktop/blob/7639b7ed0684d1883953ea931114dc19cead72a1/src/renderer/screens/settings/sections/Accounts/CurrencyRows.js#L69

Reply:
That's because we currently only show the settings for number of confirmations, we used to have other settings such as the endpoint for ripple but we don't anymore. Quoting the comment in the same linked file from @JunichiSugiura 
> // NB ideally we would have a dynamic list of settings
But we don't, so for now I think it's ok